WebRoot/css/zm.css

```diff
@@ -1985,123 +1985,193 @@ div.CompDragArea * {
+	@ContactHeaderSubTitle@
 }
 
-.ZmContactView .contactHeaderSubTable,
-.ZmContactInfoView .contactHeaderSubTable {
-	background-color:@darken(AppC,15)@;
-	border-bottom:1px solid @darken(AppC,50)@;
+.ZmContactHeaderInfo .ZmContactHeaderText:not(:empty) {
+	@ContactHeaderText@
 }
 
-.ZmContactInfoView .contactGroupTable {
-	background-color:white;
-	border:1px solid @darken(AppC,50)@;
-	float:left;
-	margin:.5em;
-	width:30%;
+.ZmContactHeaderInfo .ZmContactHeaderJobInfo:not(:empty) {
+	@ContactHeaderJobInfo@
 }
 
-.contactHeaderTags {
-	padding-top: .5em;
+.ZmContactHeaderInfo .ZmContactHeaderTags:not(:empty) {
+	@ContactHeaderTags@
 }
 
-.contactGroupTableImage {
-	vertical-align:top;
-	text-align:center;
-	padding:1em 5px 0em;
-	height:70px;
+.ZmContactHeaderRow .ZmContactHeaderLocation {
+	@ContactHeaderLocation@
 }
 
-.contactGroupTableImage IMG {
-	@ContactGroupImageRadius@
+.ZmContactHeaderLocation > TABLE {
+	@ContactHeaderLocationTable@
 }
 
-.contactGroupTableContent {
-	padding-right:.5em;
+.ZmContactHeaderLocation .ZmContactHeaderLocationIcon {
+	@ContactHeaderLocationIcon@
 }
 
-.contactGroupTableContent SPAN {
-	@NoWrap@
+.ZmContactHeaderLocation .ZmContactHeaderLocationName {
+	@ContactHeaderLocationName@
+}
+
+.ZmContactHeaderLocation .ContactHeaderLocationLabel {
+	@ContactHeaderLocationLabel@
 }
 
-.contactGroupTableContent .ImgPlus {
+/* Contact Info View Style */
+.ZmContactInfoView .ZmContactInfo:not(:empty) {
+	@ContactInfo@
+}
+
+.ZmContactInfo > TABLE {
+	@FullWidth@
+}
+
+.ZmContactInfo .ZmContactInfoMsg {
+	@ContactInfoMsg@
+}
+
+.ZmContactInfo .ZmContactInfoAction {
+	@ContactInfoAction@
+}
+
+/* Contact view info style */
+.ZmContactInfoView .ZmContactContent {
+	@ContactContent@
+}
+
+.ZmContactContent > TABLE {
+	@FullWidth@
+}
+
+.ZmContactContent .ZmContactRowWrapper {
+	@ContactRowWrapper@
+}
+
+.ZmContactRowWrapper .ZmContactRowLabel {
+	@ContactRowWLabel@
+}
+
+.ZmContactRowWrapper .ZmContactRowValueWrapper {
+	@ContactRowValueWrapper@
+}
+
+.ZmContactRowValueWrapper .ZmContacRowValue {
+	@ContactRowValue@
+}
+
+.ZmContactRowValueWrapper .ZmContacRowValueType {
+	@ContactRowValueType@
+}
+
+/* Contact Group View */
+.ZmContactInfoView .ZmContactGroupListContent {
+	@ContactGroupContent@
+}
+
+.ZmContactGroupItemTable {
+	@ContactGroupItemTable@
+}
+
+.ZmContactGroupItemTable TD {
+	@ContactGroupItemTableCell@
+}
+
+.ZmContactGroupItemImage IMG,
+.ZmContactGroupItemImage .svg-icon {
+	@ContactGroupItemImage@
+}
+
+.ZmContactGroupItemTable .ZmContactGroupItemContent * {
+	@ContactGroupItemContent@
+}
+
+.ZmContactInfoView .ZmContactGroupListContent .ZmContactGroupItemTable .ZmContactGroupItemContent {
+	@ContactGroupItemViewContentWrapper@
+}
+
+.ZmContactInfoView .ZmContactGroupListContent .ZmContactGroupItemTable .ZmContactGroupItemImage {
+	@ContactGroupItemViewImageWrapper@
+}
+
+.ZmContactView .contactHeaderRow,
+.ZmTaskEditView .contactHeaderRow {
+	height:40px;
```

Comment:
This part haven't touched now. I think it's showing as newly added, is because I have removed `.ZmContactInfoView .contactHeaderRow` and put all contact view related declarations together, which shifts this declarations down. 

Thanks